src/controllers/onrampController.ts

```
@@ -72,11 +72,9 @@ export async function registerOnRampSwap(
     }
     const parsed = bodySchema.safeParse(req.body);
     if (!parsed.success) {
-      res
-        .status(400)
-        .json({ error: "Invalid request", details: parsed.error.flatten() });
-      return;
+      throw new AppError("Invalid request", 400, "VALIDATION_ERROR", parsed.error.flatten());
```

ESLint

[error] Replace `"Invalid·request",·400,·"VALIDATION_ERROR",·parsed.error.flatten()` with `⏎········"Invalid·request",⏎········400,⏎········"VALIDATION_ERROR",⏎········parsed.error.flatten(),⏎······`

(prettier/prettier)